spec/java/util/Random.lsl

```diff
@@ -166,30 +190,45 @@ automaton RandomAutomaton: Random
     }
 
 
-    fun nextBytes (@target self: Random, arg0: array<byte>): void
+    fun nextBytes (@target self: Random, bytes: array<byte>): void
+    {
+        // #problem: is there a more efficient way?
```

Comment:
Maybe use here SYMBOLIC_ARRAY ? Or it isn't effective ?

Reply:
I expect something that modifies the provided patch of memory, not to change a local reference.